sklearn/manifold/mds.py

```diff
@@ -330,6 +331,12 @@ class MDS(BaseEstimator):
             Pre-computed dissimilarities are passed directly to ``fit`` and
             ``fit_transform``.
 
+    inductive : boolean, optional, default: False
```

Comment:
Perhaps we should have a `method` or `algorithm` or `solver` parameter rather than inductive. Remind me: do we get the same embedding of the training data, or an embedding with similar properties, either way? Does one or the other have higher computational costs?

Reply:
It's a different embedding.   The new algorithm (which can be applied to out-of-sample points) is faster, but there is an increased memory requirement as we store a matrix and I believe the stress value may not be quite as good.